### PR TITLE
wallet:list

### PR DESCRIPTION
**Pull request by lwisne**
## Summary
Goes along with https://github.com/iron-fish/ironfish/pull/2947
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: docs/onboarding/wallet.md [hunk 1]
```diff
@@ -85,11 +85,11 @@ ironfish wallet:which
 ### Accounts list
 To see the list of accounts on your node
 ```sh
-ironfish wallet
+ironfish wallet:list
```
Comment (hughy) [inline docs/onboarding/wallet.md:88]: Isn't it `wallet:accounts`?

File: src/theme/components/Terminal/Wallet/List.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useState } from "react";
 
 export default [
-    <span data-ty="input">ironfish wallet</span>,
+    <span data-ty="input">ironfish wallet:list</span>,
```
Comment (NullSoldier) [inline src/theme/components/Terminal/Wallet/List.js:4]:
```suggestion
    <span data-ty="input">ironfish wallet:accounts</span>,
```

File: docs/onboarding/wallet.md (hunk 1 above)
Comment (NullSoldier) [inline docs/onboarding/wallet.md:88]:
```suggestion
ironfish wallet:accounts
```

Comment (lwisne) [reply docs/onboarding/wallet.md]: yes. ugh

File: src/theme/components/Terminal/Wallet/List.js (hunk 2 above)
Comment (lwisne) [reply src/theme/components/Terminal/Wallet/List.js]: fixed

**Review by hughy — APPROVED** [review]

**Pull request merged**